packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,205 @@
+
+  .hds-side-nav__header-logo {
+    margin-right: auto; // Q: Make more generic?
+  }
+
+  // TEMP HACK ONLY! (TODO: replace)
+  .hds-disclosure__toggle:not(:hover) * {
+    color: var(--token-side-nav-foreground-primary) !important;
+  }
+}
+
+.hds-side-nav__header-logo {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 48px;
+  height: 48px;
+  padding: 4px;
+  color: var(--token-side-nav-foreground-primary);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &:focus,
+  &.mock-focus {
+    @include hds-focus-ring-with-pseudo-element();
+  }
+}
+
+// Some styles in cloud-ui, interfere so adding additional selector to increase specificity
+body .hds-side-nav__header-logo {
+  &:hover,
+  &.hover {
+    background: var(--token-side-nav-surface-hover);
+    border-color: transparent;
+  }
+}
+
+body .hds-side-nav__header-logo {
+  &:active,
+  &.active {
+    background: var(--token-side-nav-surface-active);
+    border-color: transparent;
+  }
+}
+
+.hds-side-nav__header-logo-icon {
+  width: auto;
+  height: 35px;
+}
+
+// * Sub-component: Body
+// ---------------------------------------
+.hds-side-nav__body {
+  flex: 1;
+  padding: 12px;
+  overflow: auto;
+}
+
+.hds-side-nav__section {
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  + .hds-side-nav__section { margin-top: 28px; }
+}
+
+.hds-side-nav__section-title {
+  padding: 0 var(--token-side-nav-item-padding-horizontal);
+  color: var(--token-side-nav-foreground-faint);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  + .hds-side-nav__list { margin-top: 12px; }
+}
+
+.hds-side-nav__list {
+  margin: 0;
+  padding: 0;
+}
+
+.hds-side-nav__list-item {
+  list-style-type: none; // Q: Should ul have role="list" added for accessibility?
+
+  + .hds-side-nav__list-item { margin-top: 4px; }
+}
+
+// generic item
+.hds-side-nav__item {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  min-height: 2.25rem;
+  padding: 0 var(--token-side-nav-item-padding-horizontal);
+  color: var(--token-side-nav-foreground-primary);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+}
+
+// interactive "link" item
+.hds-side-nav__item--link {
+  text-decoration: none;
+  overflow-wrap: break-word;
+  border-color: transparent;
+  border-radius: 5px;
+
+  .hds-badge,
+  .hds-badge-count {
+    margin-left: 8px;
+  }
+}
+
+// Some styles in cloud-ui, interfere so adding additional selector to increase specificity
+body .hds-side-nav__item--link:hover,
```

Comment:
same as above, we don't need these specificity hacks in our CSS code, you can safely remove them